src/benchmarks/micro/libraries/System.Runtime/Perf.String.cs

```diff
@@ -20,6 +20,12 @@ public static IEnumerable<object> TestStringSizes()
             yield return new StringArguments(1000);
         }
 
+        [Benchmark]
+        [Arguments(1)]
+        [Arguments(10)]
+        [Arguments(100)]
```

Comment:
would a large number be worth adding? there's certainly 10K strings eg.

Reply:
Not sure if that would be worth adding. The typical use case for this ctor is to add whitespace padding to a line of text. Example: `string combined = new string(' ', indentationLevel * 4) + originalLine;` Realistically, this value should never get above a few dozen or so.

Reply:
fair enough